dspace/config/modules/discovery.cfg

```diff
@@ -22,3 +22,5 @@ discovery.index.projection=dc.title,dc.contributor.*,dc.date.issued
 # 1) you need to set the DiscoverySearchRequestProcessor in the dspace.cfg 
 # 2) to show facet on Site/Community/etc. you need to add a Site/Community/Collection
 #	 Processors plugin in the dspace.cfg
+
+#discovery.autoReindex = false
```

Comment:
Please add a comment to describe this new setting.  We always try to document settings in the config file itself via a comment.

Reply:
Sorry Tim! Corrected: https://github.com/DSpace/DSpace/pull/2184/commits/e744d7731acf3f008ed678a7ac837df60de7180b